### PR TITLE
[WIP] Establish Cantaloupe endpoint

### PR DESCRIPTION
**Pull request by aaron-collier**
This is based off of a rebased version of @jcoyne iiif_endpoint branch.

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-  driver: djatoka
+  djatoka_url: 'http://localhost:8080/resolver'
+  driver: remote_iiif
   djatoka:
+    image: DjatokaImage
+    info: DjatokaInfoService
+  remote_iiif:
     image:
-      implementation: DjatokaImage
+      immplmentation: IiifImage
```
Comment (jcoyne) [inline config/settings.yml]: Typo.

File: config/settings.yml [hunk 2]
```diff
@@ -1,13 +1,17 @@
 stacks:
   storage_root: /stacks
-  driver: djatoka
+  djatoka_url: 'http://localhost:8080/resolver'
```
Comment (jcoyne) [inline config/settings.yml]: Shouldn't these be moved into `djatoka.info.attributes`?

File: config/settings.yml [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+  driver: remote_iiif
   djatoka:
+    image: DjatokaImage
```
Comment (jcoyne) [inline config/settings.yml]: shouldn't these be in `djatoka.image.implementation` and ``djatoka.info.implementation`?

**Comment by jcoyne** [comment]
I believe that #210 has a more correct rebase.  Do you want to rebase this branch against that one?

**Comment by aaron-collier** [comment]
Closing in favor of #210

**Pull request closed without merging**